### PR TITLE
avoid selenium detection on login, formatting of README.md, instructions for VSC

### PR DESCRIPTION
**Pull request by denisergashbaev**

### CODE REVIEW

File: kleinanzeigen.py [hunk 1]
```diff
@@ -417,14 +422,27 @@ def session_create(config):
...
+    # add this option otherwise login is blocked when using selenium 
+    # https://stackoverflow.com/a/60649888/256002
+    # options.add_argument("disable-blink-features=AutomationControlled")
+
     if config.get('headless', False) is True:
         log.info("Headless mode")
         options.add_argument("--headless")
 
     if config.get('webdriver_enabled') is False:
         options.set_preference("dom.webdriver.enabled", False)
```
Comment (neon-dev) [inline kleinanzeigen.py:435]: Setting `dom.webdriver.enabled` to `False` here was working for me without issues. What do the new options below do? Also `dom.webdriver.enabled` is now set twice with different conditions.

Comment (denisergashbaev) [reply kleinanzeigen.py:432]: I have rechecked it, true. Removed unneeded directives and added a fix for python 3.6 and older with date formatting instead

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,14 @@
+astroid==2.4.2
```
Comment (denisergashbaev) [inline requirements.txt:1]: requirements.txt as generated by (python3.6) `pip freeze > requirements.txt`

File: kleinanzeigen.py (hunk 1 above)
Comment (denisergashbaev) [reply kleinanzeigen.py:429]: @neon-dev could you please merge? I'd like to dockerize the project in my next pull request

Comment (neon-dev) [reply kleinanzeigen.py:429]: I'm just a past contributor so I don't have write access, sorry. @donwayo has to merge this

**Review by donwayo — APPROVED** [review]
Thanks for the contribution. Looks good to merge!

**Comment by denisergashbaev** [comment]
@donwayo would it be ok if I dockerize the project (and bump up the version of python to 3.8-3.9)?

**Comment by donwayo** [comment]
That sounds like a great idea. I'm eagerly waiting for the pull request :)

Eduardo Castellanos N.

**Pull request merged**